README.md

```diff
@@ -150,6 +151,40 @@ tilesets delete-source
 
 Permanently delete a tileset source and all of its files. This is not a recoverable action!
 
+### estimate-area 
+
+```shell
+tilesets estimate-area <features> -p/--precision <precision>
+```
+
+Estimates area (km<sup>2</sup>) of features with a specified precision. Accepts line-delimited GeoJSON or GeoJSON feature collections as files or via `stdin` or a list of string-encoded coordinate pairs of the form "[lng, lat]", or "lng, lat", or "lng lat". Precision must be 10m, 1m, 30cm or 1cm.
+
+Flags:
+
+* `--no-validation` [optional]: do not validate source data locally before area calculation
+* `--force-1cm` [optional]: enables 1cm precision 
```

Comment:
What does "forcing 1cm precision" mean?

Reply:
it's a flag you have to put to do 1cm precision, it'll throw an error if you choose 1cm without it. Should I add it to the description? I didn't want to make it too wordy

Reply:
That's good to know! Brainstorming some ideas:

> enables 1cm zoom calculations. The 1cm (zoom >16) precision is not available in the Tilesets API by default. Reach out to support to enable this feature.
> link: https://support.mapbox.com/hc/en-us/requests/new?ticket_form_id=360000291231

Grabbed this from the docs at https://docs.mapbox.com/mapbox-tiling-service/reference/#zoom-levels